### DISCUSSION TITLE
Change S3 target for etcd snapshot backups

### DISCUSSION THREAD

**Discussion by cro** [kube-hetzner/terraform-hcloud-kube-hetzner#695] [Q&A] [answered]
Greetings,

Is there a way to change the S3 target of the etcd snapshot backups after cluster creation?  I changed it in the kube.tf and re-ran `terraform apply` but it said it did not need to make any changes.

Thanks!
--cro

**Comment by mysticaltech** [comment, marked as answer]
@cro Not at this moment unfortunately, as this does not touch the Hetzner infra, but the k3s config file itself. You need to SSH into your control planes nodes (one after another, see readme), and edit the `/etc/rancher/k3s/config.yaml` file to put your new value there.
